include/TI/tivx_obj_desc.h

```diff
@@ -321,8 +330,11 @@ typedef struct _tivx_obj_desc_t {
     /*! \brief holds the CPU ID of the OpenVX host for this object descriptor */
     volatile uint32_t host_cpu_id;
 
+    /*! \brief ID of supplementary data object descriptor in shared memory */
+    volatile uint16_t supp_data_ID;
```

Comment:
Can we change the case of "ID" to lower case?

Reply:
done

Reply:
updated now